public/javascripts/autofill-profile-form.js

```diff
@@ -1,10 +1,11 @@
 /* eslint-disable no-unused-vars */
 let countrySelect = document.getElementById("country");
-const userCountry = document.getElementById("user-country").innerText;
+let userCountry = document.getElementById("user-country-name");
```

Comment:
What's the reason changing behind changing from const to let?

Reply:
No particular reason, think it was just a brainfart. Gonna change it back to const.